Docs/Configuration.tex

```diff
@@ -1576,12 +1576,12 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
 
   Some modern firmware, including on virtual machines such as VMware,
   may update pointers to UEFI services during driver loading and related actions.
-  Consequentially this directly breaks other quirks that affect memory management,
+  Consequently, this directly breaks other quirks that affect memory management,
```

Comment:
Any reason to change this?

Reply:
"Consequentially" is technically correct but is not usual in such situations.

Reply:
Yes, 'consequentially' is an adverb meaning 'importantly' (i.e. having (implicitly, important) consequence), 'consequently' is a conjunction meaning 'as a consequence of (this)'!

Reply:
> "Consequentially" is technically correct but is not usual in such situations.

With respect, I believe it is _not_ even technically correct here, actually.

Reply:
> With respect, I believe it is not even technically correct here, actually.

It could also have said "Importantly, this does XYZ".

Anyway, best to focus on pressing forward.

Reply:
Okay, agreed, so then it depends on whether the original meant "As a matter of great importance, ..." (consequentially), or "Because of the above, ..." (consequently). I think we're agreed on that? Vit can be more certain than us on what he original was meant to mean..!